app/components/settings/categories/display/ThemeThumbnail.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+type Props = {
+  themeVars: Object
+};
+
+export default class ThemeThumbnail extends Component<Props> {
```

Comment:
I think you can clean this up by saving this as an `svg` file and then applying the color transformation using css `style` overrides in the Javascript. Maybe not a big deal though?